actiontext/lib/action_text/engine.rb

```diff
@@ -37,21 +37,24 @@ def to_trix_content_attachment_partial_path
     end
 
     initializer "action_text.helper" do
-      ActiveSupport.on_load(:action_controller_base) do
-        helper ActionText::Engine.helpers
+      %i[action_controller_base action_mailer].each do |abstract_controller|
+        ActiveSupport.on_load(abstract_controller) do
+          helper ActionText::Engine.helpers
+        end
```

Comment:
Theoretically, this PR could also fix #38880 if `ActionController::API` also supported `helper`.